source/configuration/modules/omhttpfs.rst

```diff
@@ -0,0 +1,54 @@
+omhttpfs: Hadoop HTTPFS Output Module
+============================================
+
+===========================  ===========================================================================
+**Module Name:**             **omhttpfs**
+**Available Since:**         **8.10.0**
+**Author:**                  `sskaje <https://sskaje.me/2014/12/omhttpfs-rsyslog-hdfs-output-plugin/>`_ <sskaje@gmail.com>
+===========================  ===========================================================================
+
+This module is an alternative to omhdfs via `Hadoop HDFS over HTTP <http://hadoop.apache.org/docs/current/hadoop-hdfs-httpfs/index.html>`_.
+
+**Dependencies**
+* libcurl
```

Comment:
@sskaje Thanks for contributing! There's a tiny error in that there's a missing blank like between "Dependencies" and "libcurl" which makes it look a little weird. Could you add it and then I would merge this for the next release? Thanks again!